devtools/platform-descriptor-json/src/main/resources/codestarts/amazon-lambda-example/codestart.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: amazon-lambda-example
+ref: amazon-lambda
+type: example
+language:
```

Comment:
```suggestion
missing-languages:
  - kotlin
  - scala
language:
```

Reply:
I am not sure about the different codestarts you added, but it is really important that you add this section `missing-languages` when some languages are missing. It will make sure that this example is skipped when those languages are selected.

I also need to add a `missing-builtool`, but I'll fix that in another PR

Reply:
Those languages may work, but I never used either thus there is no example.  That's for another PR I guess.

Reply:
ok, this flag is not clear enough, I will rename it in another PR.

For now, when there is no example for a language, then you should add it to the `missing-languages` list. This is to avoid having the `base` part applied without any code, you get what I mean?